models/schema/v1/workload/secret/secret.k

```diff
@@ -1,11 +1,12 @@
 import regex
 
 schema Secret:
-    """ Secret can be used to store sensitive data.
+    """ Secrets are used to provide data that is considered sensitive like passwords, API keys,
+    TLS certificates, tokens or other credentials.
 
     Attributes
     ----------
-    $type: str, default is opaque, required.
+    $type: str, default is Undefined, required.
```

Comment:
Form the K8s reference link, an empty string is treated as an Opaque type. Why should we change the default value from Opaque to Undefined?

Reply:
remove the K8s ref link, secret type in catalog is not equal to K8s secret, so no need to keep Opaque as default value.